src/components/accounts/two_factor/TwoFactorVerifyModal.js

```diff
@@ -68,7 +68,7 @@ const TwoFactorVerifyModal = ({ open, onClose }) => {
     }
     
     const handleCancelClose = () => {
-        Mixpanel.track("2FA Modal Cancel verification")
```

Comment:
not sure why this change, is `handleCancelClose` called in some other case than dismissing 2FA modal?

Reply:
reverted